README.md

```diff
@@ -20,12 +24,23 @@ Two experiments are carried out:
         + Bruteforce messaging with RTC (run time compilation)
         + Spatial3D messaging 
         + Spatial3D messaging with RTC (run time compilation) 
+
+[![Fixed Density Benchmark](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--all.png)](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--all.png)
+[![Fixed Density Benchmark Zoomed](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--zoomed.png)](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/fixed-density--volume--step-s--model--zoomed.png)
+
+### Variable Density Benchmark
+
 + Variable Density
     + Communication Radius is fixed to `2.0`
     + Agent Density is varied per unit of volume, from `1` to `4`
     + Environment Volume is varied upto `~ 500000` units of volume.
     + A single implementation is benchmarked
         + Spatial3D messaging with RTC (run time compilation)
+		
+[![variable-density volume](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/variable-density--volume--step-s--density--3drtc.png)](sample/figures/v100-470.82.01/alpha.2-v100-11.0-beltsoff/variable-density--volume--step-s--density--3drtc.png)
+
+### Variable Communication Radius Benchmark
+
 + Variable Communication Radius
```

Comment:
The top level list item is redundent.

I can't get the github suggestion multiline to work here though (sorry).